File: ts/wam/2958.ts
export const WamTestAnonymousDailyId = 2958 as const

export interface TestAnonymousDailyIdEvent {
  /** field 1, wire `ps_test_enum_field` */
  psTestEnumField?: PSTESTENUMFIELD
  /** field 2, wire `ps_test_float_field` */
  psTestFloatField?: string
}

File: go/wam/2958.go
const WamTestAnonymousDailyId = 2958 // channel: private

type TestAnonymousDailyIdEvent struct {
	PsTestEnumField PSTESTENUMFIELD `wam:"ps_test_enum_field"` // field 1
	PsTestFloatField string `wam:"ps_test_float_field"` // field 2
}